builtins/amp-img.js

```diff
@@ -47,14 +47,21 @@ export class AmpImg extends BaseElement {
   }
 
   /** @override */
-  attributeChangedCallback(name, unusedOldValue, unusedNewValue) {
-    if (name === 'src') {
-      this.srcset_ = srcsetFromElement(this.element);
-      this.updateImageSrc_();
-    } else if (this.img_ && ATTRIBUTES_TO_PROPAGATE.indexOf(name) >= 0) {
-      this.propagateAttributes(name, this.img_,
-          /* opt_removeMissingAttrs */ true);
-    }
+  mutatedAttributesCallback(mutations) {
+    mutations.forEach(mutation => {
```

Comment:
No, I mean, why does it matter what attributes changed? Let's just propagate. We could do an `#find` if creating a new srcset is expensive.